app/code/core/Mage/Adminhtml/Helper/Config.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * OpenMage
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ * @copyright  Copyright (c) 2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Default config helper
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ * @author     Magento Core Team <core@magentocommerce.com>
```

Comment:
Is the author correct? Also, that email no longer work.

Reply:
we have the same on every file :-\ I don't remember if we did decide something about new files :-\

Reply:
@justinbeaty after writing command for updating docblocks ... :heart: 

... is it possible to extend your script to replace headers for all added files since last release to change the "author"?

Reply:
> I don't remember if we did decide something about new files :-\

We already had a discussion where someone wanted to add his company name ... cant find it atm. 

I'd prefer to have something like _OpenMage Developers_ ...

Reply:
this should be discussed in a proper discussion and/or rfc, I wouldn't block this PR which was supposed to be simple.

Reply:
also, we should ask adobe to speak about this, we shouldn't change the copyright but it doesn't make sense to keep these one...